Dockerfile

```
@@ -43,34 +43,46 @@
 # Declare runtime environment variables
 ARG NOTIFIER_API_TOKEN
 ENV NOTIFIER_API_TOKEN=$NOTIFIER_API_TOKEN
-ENV RELEASE_CONFIG=/app
+ENV CONFIG_PATH=/app/etc
 
-# Install runtime dependencies
+# Install runtime dependencies (including PostgreSQL client tools)
 RUN apt-get update -y && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)